index.js

```diff
@@ -1,88 +1,125 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+var counter = 1
+console.log("%cQuestion 1", "color: red") 
+console.log(counter) 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+
+// => 2
+var counter = 2
+console.log(counter)
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+
 
-// console.log(name)
+const name = 'Henry Velasquez'
+console.log(name)   // WHY DOES NAME HAVE A STRIKETHROUGH?
 // => "Raffy"
-// name = "Not Raffy" 
+
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
+function drinkWater(currentThirstLevel) { 
```

Comment:
Great 👍  One way to condense this and not lose any readability would be to consolidate lines 36 and 38 to:
```js
return currentThirstLevel - 1
```
A- If we don't have the `return` and we try to use this function for getting a value, it will not work as expected as demonstrated in the following example:
```javascript
let myThirstLevel = drinkWater(7)
console.log(myThirstLevel) // undefined
```
- if we do have the `return` though, the same exact code will yield different results as shown below:
```javascript
let myThirstLevel = drinkWater(7)
console.log(myThirstLevel) // 6
```